moja-kuhinjica-frontend/src/components/modal/login/LoginModal.tsx

```diff
@@ -102,6 +108,17 @@ export const LoginModal = ({
                             required: t('Šifra je obavezna.'),
                         }}
                     />
+                    <div className="remember-me">
+                        <input
+                            id="cb1"
+                            checked={rememberMe}
+                            onChange={() => {
+                                setRememberMe(!rememberMe)
+                            }}
```

Comment:
if eslint allows you, this is nicer

`onChange={() => { setRememberMe(!rememberMe) }`